src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -377,11 +376,28 @@ public Path getJarPath(String jarName) {
   }
 
   /**
-   * Checks whether the Cloud SDK Path with is valid.
+   * Checks whether the Cloud SDK path and version are valid.
    *
-   * @throws CloudSdkNotFoundException when the Cloud SDK is not installed where expected
+   * @throws CloudSdkNotFoundException when an up-to-date Cloud SDK is not installed where expected
    */
-  public void validateCloudSdk() throws CloudSdkNotFoundException {
+  public void validateCloudSdk() {
+    validateCloudSdkLocation();
```

Comment:
I'd have to dig it out, but my recollection is that convention for runtime exceptions is no throws clause but an @throws doc comment